src/test/java/com/stripe/model/StandardizationTest.java

```diff
@@ -64,6 +64,10 @@ public void allNonDeprecatedMethodsTakeOptions() throws IOException, NoSuchMetho
 				if (method.getName().startsWith("get")) {
 					continue;
 				}
+				// Skip internal methods
+				if (method.getName().startsWith("_")) {
```

Comment:
Cobertura instruments the code to generate the coverage reports. Rather than targeting Cobertura's methods specifically, I opted to simply ignore all methods that start with an underscore -- the assumption being that such methods are very likely internal and out of scope of the standardization test.

Reply:
Cool. Works for me. Thanks for explaining!